lib/rules/no-bitwise.js

```diff
@@ -36,12 +38,21 @@ module.exports = function(context) {
     }
 
     /**
+     * Checks if exceptions were provided, e.g. `{ allow: ['~', '|'] }`.
+     * @param   {ASTNode} node The node to check.
+     * @returns {boolean} Whether or not the node has a bitwise operator.
+     */
+    function allowedOperator(node) {
```

Comment:
Decided to stick with the method signature already present in this rule, but if you feel strong about it I don't mind changing.